providers/standard/src/airflow/providers/standard/operators/python.py

```diff
@@ -690,6 +695,7 @@ def __init__(
         expect_airflow: bool = True,
         skip_on_exit_code: int | Container[int] | None = None,
         index_urls: None | Collection[str] | str = None,
+        index_urls_from_connection_ids: list[str] | None = None,
```

Comment:
The other is `Collection[str]`can you make it consistent?
```suggestion
        index_urls: None | Collection[str] | str = None,
        index_urls_from_connection_ids: None | Collection[str] | str = None,
```

Reply:
Done 👍